tests/integration_tests/performance/network_tcp_throughput_test_config.py

```diff
@@ -0,0 +1,181 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Configuration file for the network TCP throughput test."""
+
+DEBUG = False
+
+IPERF3 = "iperf3"
+THROUGHPUT = "throughput"
+THROUGHPUT_TOTAL = "total"
+DURATION = "duration"
+DURATION_TOTAL = "total"
+RETRANSMITS = "retransmits"
+RETRANSMITS_TOTAL = "total"
+CPU_UTILIZATION_HOST = "cpu_utilization_host"
+CPU_UTILIZATION_GUEST = "cpu_utilization_guest"
+BASE_PORT = 5000
+
+CPU_UTILIZATION_VMM = "vmm"
+CPU_UTILIZATION_VCPUS_TOTAL = "vcpus_total"
+CONFIG = {
+    "time": 20,  # seconds
+    "load_factor": 1,
+    "modes": {
+        "g2h": [""],
+        "h2g": ["-R"],
+        "bd": ["", "-R"]
+    },
+    "protocols": [
+        {
+            "name": "tcp",
+            "omit": 5,
+            "window_size": [16, None],
+            "pkt_size": [None, 1024],
```

Comment:
I don't get what you did now. Is really important to test with small packets... My suggestion was to keep old values and add a None...

Reply:
The previous packet length was 128K, which is the default packet length used by iperf3 for TCP packets. So there is no change in terms of packet size. The value for 256K related to window size, which I've replaced with `None`, is very similar to the results obtained by using the default window size (or more specifically, as TCP decides). Please let me know if I misunderstood something.